### ISSUE TITLE
questions about mIoU on semanticKITTI

### ISSUE THREAD

**Issue by xiaoyao3302** [saltoricristiano/cosmix-uda#17]
Hi! Thanks for the awesome work! Your work really inspired me a lot.
Currently, I'm following your work and carrying on my project on cross-domain LiDAR segmentation, when I was running your code, I noticed that when I used 1/10 SynLiDAR training data (setting the version as 'mini', around 2000 point clouds) for 100 epochs to train a source-only model and tested the model on semanticKITTI, the performance was already higher than the reported source-only result of 20.4. When I trained the model for 10 epochs, the performance was bad. In addition, I noticed that you only trained the model for 10 epochs (as reported in the paper), while the training epoch was set as 100 in the config file. I wonder if the 10 training epoch was a clerical error.
I don't know if it's because I don't fully understand your work. If there is something wrong with my understanding, I would greatly appreciate you pointing it out for me.
Thanks again!

**Comment by saltoricristiano** [comment]
Hi @xiaoyao3302, 

Thanks for your interest in our paper!

We pre-trained the model for 10 epochs, as reported in the paper. I'm sorry for the confusion.
The 20.4 is taken from the paper of SynLiDAR and our source pre-training uses the warm-up strategy which is part of the CoSMIX framework. This may be the cause of your better results. 
Unfortunately, we have never played with the mini split, unless not for local debugging.

I hope this answers your doubts! I will close the issue, but feel free to re-open it!
